manifests/m/Microsoft/DSC/Preview/3.1.0.0/Microsoft.DSC.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Microsoft.DSC.Preview
+PackageVersion: 3.1.0-preview.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://github.com/PowerShell
+PublisherSupportUrl: https://github.com/PowerShell/DSC/issues
+PackageName: DSC v3
```

```suggestion
PackageName: DSC v3 Preview
```

This is needed so that the CLI can distinguish between the installations of DSC stable and DSC preview